### ISSUE TITLE
Scalar span cannot be underlined or used inline

### ISSUE THREAD

**Issue by petrcermak** [catapult-project/catapult#1371]
![screenshot from 2015-08-26 12 59 17](https://cloud.githubusercontent.com/assets/2546601/9492746/483a6d5e-4bf2-11e5-8114-187a499bc6c4.png)

@natduca: Could you please have a look at this since it's related to units, which you have touched recently?

**Comment by benshayden** [comment]
Can you set contentTextDecoration?

**Comment by petrcermak** [comment]
Yes, I can use <tt>element.contentTextDecoration = 'underline'</tt>, but I can't use it to get the span to be displayed *inline*. Besides, I don't think this approach is right because it requires "manually" copying the style of the analysis link into the scalar span.

As I said before, I think that **_unit_ formatting and _visual_ formatting should be separate** (either by nesting two elements, or defining two separate elements) so that scalar spans could be used in non-tabular environments seamlessly.

**Comment by natduca** [comment]
@petrcermak is this still an issue? I seem to have dropped the ball here. :/
